docs/pages/blog/mui-x-sep-2024-price-update.md

```diff
@@ -2,57 +2,74 @@
 title: Upcoming changes to MUI X pricing in 2024
 description: Check out the new pricing updates and how to transition to the new model.
 date: 2024-08-01T08:00:00.000Z
-authors: ['josefreitas']
+authors: ['josefreitas', 'oliviertassinari']
 tags: ['MUI X', 'Product']
 manualCard: true
 ---
 
-We're excited to share some important updates to our product offerings and pricing structure. This change reflects the expanded feature set and helps us continue delivering high-quality components to you and your users.
```

Comment:
Wasn't sure about "excited". It's mostly about pricing and an increase, not sure exactly that users would be excited.